evennia/settings_default.py

```diff
@@ -1167,7 +1171,10 @@
 # to tons of classes. You generally shouldn't need to touch protocols.
 BASE_SESSION_CLASS = "evennia.server.session.Session"
 
-# Telnet Protocol inherits from whatever above BASE_SESSION_CLASS is specified.
+# The base Portal Session class, which by default inherits from BASE_SESSION_CLASS.
+BASE_PORTAL_SESSION_CLASS = "evennia.server.portal.portalsession.PortalSession"
```

Comment:
No, because there is only ONE Server Session class for ALL Protocol Types... that wouldn't make any sense.

SERVER:
BASE_SESSION_CLASS -> SERVER_SESSION_CLASS

PORTAL:
BASE_SESSION_CLASS -> BASE_PORTAL_SESSION_CLASS -> <specific_protocol_class>